dapr/actor/runtime/config.py

```diff
@@ -71,28 +141,40 @@ def __init__(
                 If not provided, reentrancy is diabled.
             reminders_storage_partitions (int): The number of partitions to use for reminders
                 storage.
+            actor_type_configs (List[ActorTypeConfig]): Configure the behavior of specific
+                actor types.
         """
-        self._entities: List[str] = []
+        self._entities: Set[str] = set()
         self._actor_idle_timeout = actor_idle_timeout
         self._actor_scan_interval = actor_scan_interval
         self._drain_ongoing_call_timeout = drain_ongoing_call_timeout
         self._drain_rebalanced_actors = drain_rebalanced_actors
         self._reentrancy = reentrancy
         self._reminders_storage_partitions = reminders_storage_partitions
+        self._entitiesConfig: List[ActorTypeConfig] = actor_type_configs
```

Comment:
Why can't this be a `Set`? As in specifically avoiding the possibility of duplicate `ActorTypesConfig`s

Reply:
It doesn't need to be one since you always need to specify the entire List anyway.

Also `{ActorTypeConfig('hello', remindersStoragePartitions=1), ActorTypeConfig('hello', remindersStoragePartitions=2)}` would still be a set of 2 items.

We only allow the entire list to be set or update all at once. So there is little risk in inserting duplicates.